config/webpack/environment.js

```diff
@@ -1,3 +1,5 @@
 const { environment } = require('@rails/webpacker')
+const typescript =  require('./loaders/typescript')
```

Comment:
wow! it's cool to see code I contributed to [webpacker](https://github.com/rails/webpacker/pull/1145) actually being used!!